es6-destructuring/index.html

```diff
@@ -12,16 +12,16 @@
 };
 
       const { title, author, libraryID } = book1;
-      console.log('The title of the book is,', title, 'the author is ' + author + ', and the library id is', libraryID);
+      console.log(`The title of the book is, ${title}, the author is ${author}, and the library id is, ${libraryID}`);
```

Comment:
I just noticed that you used `,` in some of the places in your prior `console.log`. I keyed in on the `+`s. It's certainly legitimate to do it all with `,`s.